var/spack/repos/builtin/packages/openfoam-com/package.py

```diff
@@ -659,6 +663,10 @@ def __init__(self, spec, **kwargs):
         # spec.architecture.target is like `uname -m`
         target   = spec.architecture.target
 
+        # Platform detects as 'cray' - treat like 'linux'
+        if platform == 'cray':
+            platform = 'linux'
```

Comment:
@olesenm CrayPE makes everything ugly. I'm not sure I fully understand the situation, as I'm not familiar with the openfoam/wmake build process but I take it wmake rules need to be `linux64Gcc` when on an x86 Cray?

To my knowledge, there are no x86-based Crays that are not also 64 bit so @YtterbiJum's code block above makes sense to me. But it should be noted that Cray does make ARM systems too with the same approach for cross-compiling using their wrappers `cc`, `CC`, and `ftn` although the current Spack Cray platform was not written to with non-x86 targets in mind - it only tries to load the right craype target environment modules and uses the cray wrappers as the compilers.